DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Model/DicomDataModel.cs

```diff
@@ -0,0 +1,120 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Dicom;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors.Model
+{
+    public class DicomDataModel
+    {
+        public static HashSet<DicomVR> CryptoHashSupportedVR
+        {
+            get
+            {
+                return new HashSet<DicomVR>()
```

Comment:
**[Suggestion]**
The HashSet will be created every time the get method is called.
Change it to a static member and just return the instance here.
I noticed it's used in the initialization of a static member of other classes.
But if other developers use this in the future, they probably use it as a normal property.

Same suggestion to other fields.